### PR TITLE
Use cookie friendly domains (Users)

### PR DESCRIPTION
**Pull request by GuillaumeCisco**

### CODE REVIEW

**Comment by ClementGautier** [comment]
@GuillaumeCisco Please fix the conflicts before I review

**Comment by GuillaumeCisco** [comment]
This is the third time I rebase this branch after repo-renaming, hope it works now.

File: Jenkinsfile [hunk 1]
```diff
@@ -164,14 +171,16 @@ pipeline {
...
               sh ./build-docker-images.sh
               export SUBSTRA_PATH=/tmp/substra/
               cd ./docker && python3 start.py -d --no-backup
               sleep 120
-              echo \$MY_HOST_IP owkin.substra-backend >> /etc/hosts
-              echo \$MY_HOST_IP chunantes.substra-backend >> /etc/hosts
+              echo \$MY_HOST_IP owkin.substra-backend.xyz >> /etc/hosts
+              echo \$MY_HOST_IP owkin.substra-frontend.xyz >> /etc/hosts
+              echo \$MY_HOST_IP chunantes.substra-backend.xyz >> /etc/hosts
+              echo \$MY_HOST_IP chunantes.substra-frontend.xyz >> /etc/hosts
```
Comment (ClementGautier) [inline Jenkinsfile]:
```suggestion
              echo \$MY_HOST_IP substra-backend.owkin.xyz >> /etc/hosts
              echo \$MY_HOST_IP substra-frontend.owkin.xyz >> /etc/hosts
              echo \$MY_HOST_IP substra-backend.chunantes.xyz >> /etc/hosts
              echo \$MY_HOST_IP substra-frontend.chunantes.xyz >> /etc/hosts
```

Comment (GuillaumeCisco) [reply Jenkinsfile]: thanks, I updated it

File: Jenkinsfile [hunk 2]
```diff
@@ -164,14 +171,16 @@ pipeline {
             ])
 
             sh """
+              python3 ./substrabac/node/generate_nodes.py
```
Comment (ClementGautier) [inline Jenkinsfile]:
```suggestion
              python3 ./backend/node/generate_nodes.py
```

File: Jenkinsfile [hunk 3]
```diff
@@ -164,14 +171,16 @@ pipeline {
...
             sh """
+              python3 ./backend/node/generate_nodes.py
               sh ./build-docker-images.sh
               export SUBSTRA_PATH=/tmp/substra/
               cd ./docker && python3 start.py -d --no-backup
               sleep 120
-              echo \$MY_HOST_IP owkin.substra-backend >> /etc/hosts
-              echo \$MY_HOST_IP chunantes.substra-backend >> /etc/hosts
+              echo \$MY_HOST_IP substra-backend.owkin.xyz >> /etc/hosts
+              echo \$MY_HOST_IP substra-frontend.owkin..xyz >> /etc/hosts
```
Comment (jmorel) [inline Jenkinsfile]: Extra `.`

Comment (GuillaumeCisco) [reply Jenkinsfile]: thanks a lot for catching this

**Comment by GuillaumeCisco** [comment]
I will fix the `substra-backend.owkin.xyz:8001` urls to `substra-backend.owkin.xyz:8000`
Thanks.

But I found no occurrences of `substra-backend.chunantes.xyz:8000`

**Pull request merged**